src/filefinder.cpp

```diff
@@ -920,28 +826,15 @@ FileFinder::Directory FileFinder::GetDirectoryMembers(const std::string& path, F
 		}
 	}
 
-#ifdef _WIN32
-#  undef DIR
-#  undef opendir
-#  undef closedir
-#  undef dirent
-#  undef readdir
-#endif
-#undef wpath
-#ifdef PSP2
-	closedir(dir);
-#endif
 	return result;
 }
 
-EasyRPG_Offset FileFinder::GetFileSize(const std::string& file) {
-	StatBuf sb;
-	int result = GetStat(file.c_str(), &sb);
-	return (result == 0) ? sb.st_size : -1;
+int64_t FileFinder::GetFileSize(const std::string& file) {
+	return Platform::File(file).GetSize();
 }
 
 bool FileFinder::IsMajorUpdatedTree() {
-	EasyRPG_Offset size;
+	int64_t size;
```

Comment:
This variable is uninitialized, and also seems to only be used in some conditional blocks. I would move it inside the blocks.

`auto size = GetFileSize(...)`